src/main/scala/org/broadinstitute/dsde/firecloud/MethodsClient.scala

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.dsde.firecloud
+
+import akka.actor.{Actor, ActorRef, Props}
+import akka.event.Logging
+import org.broadinstitute.dsde.firecloud.MethodsClient.MethodsListRequest
+import org.broadinstitute.dsde.firecloud.model.MethodEntity
+import org.broadinstitute.dsde.firecloud.model.MethodEntityJsonProtocol._
+import spray.client.pipelining._
+import spray.http.HttpHeaders.Cookie
+import spray.http.{HttpRequest, RequestProcessingException, StatusCodes}
+import spray.httpx.SprayJsonSupport._
+import spray.json.DefaultJsonProtocol
+import spray.routing.RequestContext
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+object MethodsClient {
+  
+  case class MethodsListRequest()
+
+  def props(requestContext: RequestContext): Props = Props(new MethodsClient(requestContext))
+
+}
+
+class MethodsClient(requestContext: RequestContext) extends Actor {
+
+  import DefaultJsonProtocol._
+  import system.dispatcher
+
+  implicit val system = context.system
+  val log = Logging(system, getClass)
```

Comment:
I tested this - by removing the akka lines in the config, that let logback take effect.